app/views/customers/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1><%= @customer.name %></h1>
+<ul>
+  <% @customer.items.each do |item| %>
+    <li><%= "#{item.name} - $#{item.price} (From: #{item.supermarket.name})" %></li>
+  <% end %>
+</ul>
+
+<%= form_with url: add_item_customer_path(@customer), method: :post, local: true, id: 'new_item' do %>
+  <label for="item_id">Item ID</label>
+  <%= text_field_tag :item_id %>
+  <%= submit_tag 'Add Item' %>
```

Comment:
This `_tag` syntax is not quite right. You'll want to review the lesson on forms to see the correct syntax.